server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/GetMailboxesMethodTest.java

```diff
@@ -208,7 +208,7 @@ public void getMailboxesShouldReturnSortOrder1000OnUnknownFolder() throws Except
     }
 
     @Test
-    public void getMailboxesShouldReturnInboxWithSortOrder10OnDifferenceCase() throws Exception {
+    public void getMailboxesShouldReturnCommonMailboxSortOrderWhenInboxOnDifferenceCase() throws Exception {
```

Comment:
I don't agree to change this test. It is to make sure Inbox always appear before other mailboxes

Reply:
It's not _Inbox_ in this test, but _InBox_

Reply:
and inbox is always case insensitive